src/dlr_tvm.cc

```diff
@@ -204,6 +205,10 @@ void TVMModel::GetOutputSizeDim(int index, int64_t* size, int* dim) {
   *size = 1;
   const DLTensor* tensor = outputs_[index];
   for (int i = 0; i < tensor->ndim; ++i) {
+    if (tensor->shape[i] < 0) {
```

Comment:
Overall, there is value in this change. Can we add unit tests for this change and add this to the documentation of the `GetOutputSizeDim()` and `GetInputDim()`